tests/base/functions_12.cc

```diff
@@ -32,7 +32,7 @@ void check()
 
   std::vector<double> coeffs(n_mon);
   for (unsigned int i = 0; i < n_mon; ++i)
-    coeffs[i] = std::pow(-1.0,i)*(i+1);
+    coeffs[i] = std::pow(-1.0,static_cast<double>(i))*(i+1);
 
```

Comment:
I didn't see this test failing, but it makes sense to change this given last week's discussion on the MAC clang compiler.